### ISSUE TITLE
Add missing translated content

### ISSUE THREAD

**Issue by cthibaultatx** [cityofaustin/techstack#211]
- [ ] ENGLISH DONE
- [ ] ARABIC DONE
- [ ] VIETNAMESE DONE
- [ ] SPANISH DONE

### Get ready for curbside bulk pickup
- [x] Update steps based on original content file (missing a step): https://docs.google.com/document/d/1hkAWkdQVluWruNyw-6u8EiUTnvzs_NixWW4YPnZ0s4s/edit?usp=sharing
- [x] replace on step #3 "My collection schedule"with a jump link to scroll down to the app
- [x] replace link on step #1 "What do I do with tool" with a jump link to scroll down to app
- [x] replace recollect single purpose instance with tabbed recollect instance so a user can both look up items and check their schedule

### Get ready for curbside compost pickup
- [x] add "what do I do with" app to page
- [x] contact has "default" in it which looks strange
- [x] country looks strange UNITED_STATES

### Pickup free paint
- [x] At the bottom of the page "Need to drop off items to be recycled? See how to drop off at the Recycle and ReUse Center." the link is broken. And this is new and needs to be translated.

### Dropoff hazardous waste
- [x] contact has "default" in it which looks strange

### Pick up Free Paint...
- [x] Replace app title "Hazardous waste dropoff center" with "Ghé thăm Cửa Hàng Tái Dụng tại Trung Tâm Tái Chế & Tái Dụng" – it's the first line of the first step
- [x] In the second paragraph of body content, the last two bullets are supposed to be descriptors of the 5th bullet. We need to separate last 2 bullets into separate paragraph with an asterisk. Add asterisk to 5th bullet item. Check the English version for accuracy.

### Drop off hazardous household waste
- [x] Use "Kiểm tra bằng công cụ “What do I do with” dưới đây để xác định những loại nào được chấp nhận" as the Vietnamese app title.
- [ ] add jump link to "what do I do with" to scroll down to app
- [x] Replace app title "Hazardous waste dropoff center" with "Ghé thăm Cửa Hàng Tái Dụng tại Trung Tâm Tái Chế & Tái Dụng" – this is the first step above ^^ so assume will be fixed once and not necessary here.

### Get ready for curbside bulk pickup
- [x] Step #7 is actually bullets for step #6 – same problem in English version
- [x] There's a step missing in both English and Vietnamese versions (see checkbox under English header)
- [ ] add jump link to "what do I do with" and "my collection schedule" to scroll down to app

### Look up your trash, recycling, and compost pickup days
- [ ] add jump link to "my collection schedule" to scroll down to app
- [x] Replace 2 body paragraphs that are currently in English with the body content in this word doc:
[Look up your trash, recycling, and compost pickup days v2_Viet.docx](https://github.com/cityofaustin/techstack/files/1838050/Look.up.your.trash.recycling.and.compost.pickup.days.v2_Viet.docx)

### Get ready for curbside compost pickup
- [x] replace "PDF link" with "Curbside Compost Service Area"
- [x] add link to http://www.austintexas.gov/department/residential-rates-fees to "vị giảm kích thước thùng xe rác xuống loại nhỏ hơn, đỡ tốn kém hơn." in body copy

### Pick up free paint...
- [x] Replace app title "Hazardous waste dropoff center" with "تفضل بزيارة مستودع إعادة الاستخدام  في مركز إعادة التدوير وإعادة الاستخدام"
- [x] Put in body content from: https://drive.google.com/file/d/1Nc4YaL32cfjDtRoCBgPdWZskrIO5hVQ2/view?usp=sharing

### Drop off household hazardous waste...
- [ ] add jump link to "what do I do with" to scroll down to app
- [x] Replace app title "Check the "what do I do with" tool below to find out what items are accepted." with "أطلع على أداة "What do I do with" أدناه لمعرفة العناصر المقبولة"
- [x] Replace app title "Hazardous waste dropoff center" with "تفضل بزيارة مستودع إعادة الاستخدام  في مركز إعادة التدوير وإعادة الاستخدام" (also a task under Pick up free paint Arabic updates ^)

### Get ready for bulk item pickup
- [x] The bullets for step #5 should be bullets under step #4. Step 4 should read:

.يرجى مراجعة قائمة العناصر كبيرة الحجم التي يمكن جمعها والعناصر التي لا يمكن جمعها أدناه. افصل العناصر إلى ثلاثة أكوام
- المعادن - تشمل الأجهزة، يجب إزالة الأبواب
- إطارات سيارات الركاب - أقصى عدد مسموح به ثمانية إطارات لكل أسرة، يجب إزالة الحواف، غير مسموح بإطارات الشاحنات أو الجرارات
- العناصر غير المعدنية - تشمل السجاد والأخشاب الخالية من المسامير

- [ ] add jump link to "what do I do with" and "my collection schedule" to scroll down to app

### Get ready for curbside compost pickup
- [x] add "what do I do with" app to page (duplicate task from English version ^)
- [ ] add link to http://www.austintexas.gov/department/residential-rates-fees to " وهذا قد يسمح لك بتصغير حجم عربة القمامة إلى حجم أصغر وبأقل تكلفة." in body copy
- [x] replace "PDF link" with "Curbside Compost Service Area"

### Look up your recycling, trash, and compost pickup days
- [ ] add jump link to "اكتب عنوان شارعك في المربع أدناه" in the first line of step #1 to scroll down to app
- [x] Replace 2 body paragraphs that are currently in English with the body content in this word doc:

## Spanish - moved to issue #217

**Comment by cthibaultatx** [comment]
@ashleecoa @courtneyjacinic @LauraTrujillo On some service pages, we say "My collection schedule" and on others we simply instruct people on what to so: "Type your street address in the box below" I would replace the copy but I'm not sure we have the translation for that so I'd rather wait on you to edit the English version before seeing what to do with translated versions.

**Comment by ifsimicoded** [comment]
More Arabic follow up -- I'm curious if the greater than arrows are expected to go the other direction (less than arrows) -- as in the arrows in the screenshots attached
![screen shot 2018-03-24 at 1 41 57 am](https://user-images.githubusercontent.com/3039125/37861466-6f23c8f6-2f08-11e8-9f31-bb303ad12e4b.png)
![screen shot 2018-03-24 at 1 41 50 am](https://user-images.githubusercontent.com/3039125/37861467-714b15bc-2f08-11e8-9229-24650dfe6d9a.png)

**Comment by mateoclarke** [comment]
@ifsimicoded I mentioned the arrows to @cthibaultatx on Friday morning. We were going to wait and see what came out of user testing.

**Comment by ashleecoa** [comment]
@cthibaultatx @ifsimicoded @mateoclarke I can also ask the translator service. We asked them about if Arabic can be centered and the answered for no charge, so a question like this wouldn't cost anything. So let me know and I can email them.

**Comment by cthibaultatx** [comment]
@ashleecoa @ifsimicoded let's circle up on this in person. Don't think we need to reach out but want to review with you first.

**Comment by ifsimicoded** [comment]
opening a new issue for jumps links
Also --> add link to http://www.austintexas.gov/department/residential-rates-fees to " وهذا قد يسمح لك بتصغير حجم عربة القمامة إلى حجم أصغر وبأقل تكلفة." in body copy
@benweatherman can't locate where in the body copy this needs to be pasted. @cthibaultatx or @ashleecoa maybe you can help out here
